modules/ec2/variables.tf

```diff
@@ -599,3 +606,8 @@ variable "vitals_tsdb_address" {
   description = "Time series database address for Vitals e.g. my-prometheus.net:9090"
   type        = string
 }
+
+variable "portal_and_vitals_key_arn" {
+  description = "ARN of the secret which contains the token used to unlock portal and vitals in Kong V3"
+  type        = string
+}
```

Comment:
this should probably have a default of `null` or `""` as portal and vitals might not be enabled

Reply:
Updated - also did that same for the ECS `variables.tf`